### PR TITLE
frdvo/c02-network05 Add Internet Gateway

### PR DESCRIPTION
**Pull request by frdvo**
# Description

Exercise (e.g. C01-E01): c02-network05

Student Name: Fernando Rolnik

> Create, tag and attach Internet Gateway

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network05/frdvo/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+#Tag Internet Gateway
+aws ec2 create-tags --resources igw-0116304685cc6f00b --tags Key=Name,Value=devopsacademy-igw
+
+#Attach Internet Gateway
+aws ec2 attach-internet-gateway --internet-gateway-id igw-0116304685cc6f00b --vpc-id vpc-0a2b7db4956438d22
+```
+Add your commands and their outputs here
+```bash
+ 🐳 :: 📦 2  📑 :: 1   👉 /home/fer/repos/academy/classes/02class/exercises/c02-network05 
```
Comment (denstorti) [inline classes/02class/exercises/c02-network05/frdvo/README.md:18]: that's a lot of info and emojis 😆 
can you just add the formatted output below the respective command just above?

File: classes/02class/exercises/c02-network05/frdvo/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Commands for creating the internet gateway:
+```
+#Create Internet Gateway
+aws ec2 create-internet-gateway
+
+#Tag Internet Gateway
+aws ec2 create-tags --resources igw-0116304685cc6f00b --tags Key=Name,Value=devopsacademy-igw
+
+#Attach Internet Gateway
+aws ec2 attach-internet-gateway --internet-gateway-id igw-0116304685cc6f00b --vpc-id vpc-0a2b7db4956438d22
```
Comment (denstorti) [inline classes/02class/exercises/c02-network05/frdvo/README.md:14]: can you run `aws ec2 describe-internet-gateways --internet-gateway-ids $IGW` after attaching and include the output here?

File: classes/02class/exercises/c02-network05/frdvo/README.md (hunk 1 above)
Comment (frdvo) [reply classes/02class/exercises/c02-network05/frdvo/README.md]: All good mate.
I'm excited using Cascadia Code PL, but I can't see some characters here. I will send clear the next PR.

File: classes/02class/exercises/c02-network05/frdvo/README.md (hunk 2 above)
Comment (frdvo) [reply classes/02class/exercises/c02-network05/frdvo/README.md]: Easy

**Pull request merged**